js/maps_lib.js

```diff
@@ -209,22 +210,37 @@
     };
 
     MapsLib.prototype.doSearch = function () {
+        console.log('doSearch>>>');
```

Comment:
Same as in line 168. Will you need this message in the console to debug future code?